document/core/exec/instructions.rst

```diff
@@ -835,7 +840,7 @@ SIMD instructions are defined in terms of generic numeric operators applied lane
 :math:`\K{i32x4.}\DOT\K{\_i16x8\_s}`
 ....................................
 
-1. Assert: due to :ref:`validation <valid-vitestop>`, two values of :ref:`value type <syntax-valtype>` |V128| is on the top of the stack.
+1. Assert: due to :ref:`validation <valid-simd-dot>`, two values of :ref:`value type <syntax-valtype>` |V128| is on the top of the stack.
 
 2. Pop the value :math:`\V128.\VCONST~c_2` from the stack.
 
```

Comment:
I can't add a comment on the correct line, but point 8. below is a typo too, it should be 7. (I wonder if there are some RST linters that can help us catch such things...)

Reply:
Fixed.